go-controller/pkg/metrics/ovn.go

```diff
@@ -279,6 +283,121 @@ func ovnControllerConfigurationMetricsUpdater() {
 	}
 }
 
+func ovnControllerServiceSubnetMetricsPublisher() {
+	var cidrV4 = ""
+	var cidrV6 = ""
+	var v4Size = 0
+	var v6Size = 0
+	for _, cidr := range config.Kubernetes.ServiceCIDRs {
+		ones, bits := cidr.Mask.Size()
+		size := bits - ones
```

Comment:
I think we want the prefix size here (i.e: `ones` only). What was the reason you performed this calculation?

Reply:
Thanks for catching this. Somehow I was thinking the size of the network in terms of how many  nodes/pods can fit in the network. Corrected.